offchain/indexer/src/main.rs

```diff
@@ -2,17 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0 (see LICENSE)
 
 use clap::Parser;
-use tracing_subscriber::filter::{EnvFilter, LevelFilter};
 
-use indexer::CLIConfig;
+use indexer::{CLIConfig, IndexerConfig};
+use logs;
+use tracing::info;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let filter = EnvFilter::builder()
-        .with_default_directive(LevelFilter::INFO.into())
-        .from_env_lossy();
-    tracing_subscriber::fmt().with_env_filter(filter).init();
+    let config: IndexerConfig = CLIConfig::parse().into();
 
-    let config = CLIConfig::parse().into();
+    logs::configure(&config.logs_config);
+
+    info!(?config, "Starting Indexer");
```

Comment:
```suggestion
```
This line looks like a duplicate of the following one:

https://github.com/cartesi/rollups-node/blob/e9a9c23299a79a294d2d5ad85099a8bcdae94bfe/offchain/indexer/src/lib.rs#L16

Reply:
@fmoura FYI